lab_1/main.py

```diff
@@ -4,6 +4,18 @@
 """
 
 
+def read_from_file(path_to_file: str) -> str:
```

Comment:
Please, do not move functions from one place to another.

Reply:
ok